src/frontend/components/SøknadsSteg/Forside/Forside.tsx

```diff
@@ -37,6 +38,26 @@ const StyledAlertStripeUtvidetInfo = styled(FamilieAlert)`
     margin-top: 3rem;
 `;
 
+const StyledAccordion = styled(Accordion)`
+    width: 100%;
+`;
+
+const Layout = styled.div`
+    max-width: var(--innhold-bredde);
+    display: grid;
+    gap: 3rem;
+    margin: 2rem auto 4rem auto;
+
+    @media all and ${device.tablet} {
+        max-width: 100%;
+        margin: 2rem 2rem 4rem 2rem;
+    }
+`;
```

Comment:
Kunne man heller ha brukt VStack her?

Reply:
Trenger man egentlig å sjekke om device er tablet her?

Reply:
Vi snakket om det! Men måtte uansett ha lagt på resten av stylingen, så gikk bort fra det 🤔 om du foretrekker at vi bruker komponenten allikevel kan vi gjøre det

Reply:
Ah, det med tablet er gammelt og bare videreført

Reply:
Ah skjønner! Men trenger vi egentlig å bruke grid her? Hvis jeg har skjønt det riktig så skal alt innholdet alltid være under hverandre, og da er vel det egentlig ikke nødvendig 🤔 Personlig tror jeg at jeg foretrekker å bruke VStack, men her er det ingen fasit, så dere kjører på med det dere syns er best 🙌

Reply:
Vi trenger å ha med bredden, så da må det i såfall bli en styled `VStack` med mindre det er noe jeg ikke har sett her 🤔

Reply:
Tror ikke jeg skjønte helt hvorfor vi ikke skulle ha device.tablet 🤔 noen som kan forklare?

Reply:
Jaa, jeg tenkte stylet VStack isåfall, og kanskje ha kalt den `StyledVStack` i stedet for `Layout`. Personlig liker jeg disse komponentene fra Aksel fordi det gjør koden lettere å lese da man slipper å bla seg opp til css-koden for å skjønne hva slags komponent det er. Da kunne man droppet `display: grid` og flyttet gap ned som prop til komponenten. Meeen gir også mening sånn dere har gjort det så med på det også hvis dere foretrekker det 😊

Reply:
Når det gjelder device.tablet var jeg bare litt usikker på hvorfor vi må ha med denne. Ville tenkt at det man ønsker egentlig er å sjekke om skjermen er smalere enn et gitt brekkpunkt. Typ om du har nettleseren veldig smal på PC ønsker vi kanskje også å ha 100% bredde? Men skjønte på svaret til Kristine at det gjøres sånn her ellers i koden, så går fint for meg å bare kjøre på med denne løsningen her også 😄